src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -3,7 +3,7 @@ antlr4-python3-runtime==4.7.2
 applicationinsights==0.11.9
 argcomplete==1.11.1
 asn1crypto==0.24.0
-azure-appconfiguration==1.1.0
+azure-appconfiguration==1.1.1
```

Comment:
pls run live test to make sure no breaking change  in sdk version bumping

Reply:
I've run all live tests and updated test recordings in this PR.